examples/31-override-bootstrap-command-unmanaged-nodegroup.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: override-unmanaged-nodegroup-bootstrap
+  region: us-west-2
+  version: '1.21'
+
+nodeGroups:
+  - name: ng-1
+    minSize: 1
+    maxSize: 2
+    ami: ami-12345
+    desiredCapacity: 1
+    iam:
+      attachPolicyARNs:
+        - arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy
+        - arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy
+        - arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
+        - arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly
```

Comment:
These policies are attached by default and are unrelated to using a custom AMI. To keep this example focused, we should remove this and consider adding another example demonstrating the `iam.attachPolicyARNs` feature. 

```suggestion
```